composer.json

```diff
@@ -20,6 +20,9 @@
         "zicht/framework-extra-bundle": "^9",
         "zicht/util": "^1.7"
     },
+    "conflict": {
+        "psr/container": ">1.1.1"
```

Comment:
In `1.1.2` is deze commit geintroduceerd https://github.com/php-fig/container/commit/6009776263dfc7aaa8559fb1b3ab30be907ad1a8#diff-b76faa5405b0efea15f9586f4d772bc5e76cf5bc0f066fa821127eca94c8f583 en PHP 7.2 vindt het niet OK als https://github.com/symfony/dependency-injection/blob/4.4/Exception/ExceptionInterface.php opnieuw `\Throwable` implementeert.

Omdat we (tijdelijk) `--ignore-platform-reqs` gebruiken bij de `composer install` komt versie `1.1.2` mee.

Reply:
Waarom gebruik je `--ignore-platform-reqs`?

Reply:
Aha, prima 👍🏻 Leef je uit! 😄